src/js/serial.js

```diff
@@ -69,13 +69,13 @@ const serial = {
                             self.error = info.error;
                             setTimeout(function() {
                                 chrome.serial.setPaused(info.connectionId, false, function() {
-                                    self.getInfo(function (info) {
-                                        if (info) {
-                                            if (info.paused) {
+                                    self.getInfo(function (getInfo) {
```

Comment:
This rename seems to make the code less intuitive and harder to read (`getInfo` would be a getter for `info`, and not for `getInfo`).

Reply:
info is already used in upper scope. Like to have an exclusive object for the function. Changed it to `_info`